types/uikit/index.d.ts

```diff
@@ -1,1441 +1,583 @@
-// Type definitions for uikit 2.27
+// Type definitions for uikit 3.3
 // Project: https://getuikit.com
-// Definitions by: Giovanni Silva <https://github.com/giovannicandido>, Ivo Senner <https://github.com/s0x>
+// Definitions by: Giovanni Silva <https://github.com/giovannicandido>
+//                 Ivo Senner <https://github.com/s0x>
+//                 Weiyu Weng <https://github.com/pcdotfan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.7
```

Comment:
I'm guessing the `Minimum TypeScript Version: 3.7` comment below is referring this? That is what it does - it requires consumers to be using at least TS3.7. Authors almost never actually want to do this...